restore_ports/restore_ports.tcl

```diff
@@ -241,6 +252,9 @@ while {[string index [lindex $::argv 0] 0] eq "-"} {
       v {
         set ui_options(ports_verbose) yes
       }
+      d {
```

Comment:
This is a duplicate case.

Reply:
...oh huh, I swear it wasn't a duplicate yet when I first opened the PR; I guess it can be removed now... do you want me to do that, or...?

Reply:
Fixed in 2ebb535758982474357b7b5a86e2dfc8a762286c. Thanks for reviewing!